scripts/config-set.js

```diff
@@ -47,6 +48,20 @@ function writeMergedSettings(mc, key, newSettings) {
     )
 }
 
+function clobberSettings(mc, key, newSettings) {
```

Comment:
Could throw in:
```
+  if (! Array.isArray(newSettings)) {
+    console.error('newSettings is not an Array', newSettings)
+  }
```
FWIW. Type checking "hashes" is left as an exercise to the reader.